### DISCUSSION TITLE
Communicates with host for which no DNS query was performed

### DISCUSSION THREAD

**Discussion by miglen** [valinet/ExplorerPatcher#1018] [General]
According to this [AlienVault submission](https://otx.alienvault.com/indicator/file/b49e85051a2ebbc52ad69978c6419296d9c8777325261c0c082cf5f3855b8b4d) the application is communicating with [64.225.91.73](https://myip.ms/info/whois/64.225.91.73) hosted in DigitalOcean hosting a bunch of websites. Can you confirm if this is an expected communication and where is it coming from?

**Comment by valinet** [comment]
All resources used by the app are either hosted by Microsoft, GitHub or Google. It does so in various places, off the top of my head:

* Updater
* Symbols
* Downloading some resources (Windows 10 Start menu related from GitHub, weather widget related from Google and GitHub)

Directly by IP? Don't think so. And where in the code, exactly? The app is fully open, the site couldn't bother with a stack trace. I mean, typical "AV vendor" fashion, didn't expect more tbh.

**Reply by miglen** [reply]
Yeah, I had a look over the source and I couldn't find this host anywhere, so assumed it could be one of these heuristic findings. Thanks, appreciate it @valinet
